ceno_emul/src/syscalls.rs

```diff
@@ -6,7 +6,7 @@ pub mod keccak_permute;
 // Using the same function codes as sp1:
 // https://github.com/succinctlabs/sp1/blob/013c24ea2fa15a0e7ed94f7d11a7ada4baa39ab9/crates/core/executor/src/syscalls/code.rs
 
-pub const KECCAK_PERMUTE: u32 = 0x00_01_01_09;
+pub use ceno_rt::syscalls::KECCAK_PERMUTE;
```

Comment:
No need to specify the magic number twice in our code, and risk it going out of sync in multiple places.